src/accordion.js

```diff
@@ -0,0 +1,9 @@
+export default function() {
+  var bar = $(`.accordion__bar`);
+
+  bar.on('click', function(ev) {
+    $(`.accordion__bar-ul`).slideUp();
```

Comment:
This is over selecting ALL accordion__bars. This causes the current item to slide back up then down when clicked, which is not the desired behavior.  We will map out the DOM as a tree to see the relations and select only the other `.accordion__bar-ul` elements.